roles/foreman_installer/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: 'Run installer'
+  shell: >
+      {{ foreman_installer_command }}
+      --scenario {{ foreman_installer_scenario }}
+      {{ (foreman_installer_verbose|bool) | ternary("-v", "") }}
+      {{ (foreman_installer_no_colors|bool) | ternary("--no-colors", "") }}
+      {{ (foreman_installer_disable_system_checks|bool) | ternary("--disable-system-checks", "") }}
```

Comment:
@evgeni In Forklift, we had included this as defined options due to our use of them in different places. I wonder if that makes sense in this use case though? When does something get a dedicated option vs. rely on the user to specify it for their case.

Reply:
Depends? ;)

I think having a flag/option for verbose is a good thing, colors and system checks maybe less so?